### DISCUSSION TITLE
Recurring Issues

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#1528] [General]
Recurring Issues

We have several tpots deployed in our environment, and we constantly encounter sudden problems that we don't know how to resolve.
Often, as seen here, the issue arises where constant notifications appear, indicating that it cannot write files.

![ProblemHoPoFSCK](https://github.com/telekom-security/tpotce/assets/78480832/776928e9-4a39-4112-8d54-2f76b91d230b)

Attempts to fix this using fsck have occasionally provided temporary relief, but in general, the tpot has had to be reinstalled.

Now, we have even received the following message, and it is no longer possible to input anything:
![HoPoRIP](https://github.com/telekom-security/tpotce/assets/78480832/6e77038a-feae-454c-a77f-d2d7647c4d8d)

Is there a way to prevent and resolve these issues?

Currently, we are still using version 22.04.
We are using an NUC 7i3BNH running ESXi 7.0.1.

We also need to disable the ddospot since otherwise, the hard drive fills up very quickly every time. However, we've been experiencing these issues like in the first screenshot even before the ddospot was introduced.

**Comment by t3chn0m4g3** [comment]
Based on the information provided it is unclear what caused the behaviour.
T-Pot 22.04 is no longer maintained and can no longer be installed, hence it is recommended to install [T-Pot 24.04.0](https://github.com/telekom-security/tpotce/releases/tag/24.04.0). Follow the [ReadMe](https://github.com/telekom-security/tpotce?tab=readme-ov-file#customize-t-pot-honeypots-and-services) to guide you through the new install procedure.
T-Pot 24.04.0 also provides the ability to [adjust T-Pot](https://github.com/telekom-security/tpotce?tab=readme-ov-file#customize-t-pot-honeypots-and-services) to the services needed / suited for your environment.
